Source/WebKit/NetworkProcess/SharedWorker/WebSharedWorkerServer.h

```diff
@@ -40,11 +40,6 @@ namespace WebKit {
 class WebSharedWorkerServer;
 }
 
-namespace WTF {
-template<typename T> struct IsDeprecatedWeakRefSmartPointerException;
-template<> struct IsDeprecatedWeakRefSmartPointerException<WebKit::WebSharedWorkerServer> : std::true_type { };
```

Comment:
Why are we removing this without making `WebSharedWorkerServer` refcounted?

Reply:
The exception can go away if an object support either CheckedPtr or RefPtr, not just RefPtr.